src/v2/style-guide/index.md

```diff
@@ -733,23 +730,23 @@ components/
 </summary>
 {% endraw %}
 
-These components lay the foundation for consistent styling and behavior in your application. They may **only** contain:
+Estas componentes son la base para la definición de estilos y comportamientos consistentes en la aplicación. Estas **solamente** deben contener:
 
-- HTML elements,
-- other base components, and
-- 3rd-party UI components.
+- elementos HTML,
+- otras componentes base, y
+- elementos UI de terceros.
 
-But they'll **never** contain global state (e.g. from a Vuex store).
+Pero, **nunca** deben contener estado global (como una _store_ Vuex por ejemplo).
 
-Their names often include the name of an element they wrap (e.g. `BaseButton`, `BaseTable`), unless no element exists for their specific purpose (e.g. `BaseIcon`). If you build similar components for a more specific context, they will almost always consume these components (e.g. `BaseButton` may be used in `ButtonSubmit`).
+Sus nombres generalmente incluyen el nombre del elemento que envuelven(por ejemplo `BaseButton`, `BaseTable`), a menos que no exista un elemento para su propósito específico (por ejemplo `BaseIcon`). Si usted construye componentes similares para un contexto más específico, ellas casi siempre consumirán estas componentes (por ejemplo `BasButton` podría ser consumido en `ButtonSubmit`).
```

Comment:
- Please, add a space at `envuelven (por ejemplo`
- Plase, change `ellas casi siempre consumirán estas componentes` to `casi siempre consumirán estos componentes` or something like that.